azure/table_azure_cosmosdb_mongo_collection.go

```diff
@@ -0,0 +1,366 @@
+package azure
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/cosmos-db/mgmt/2021-04-01-preview/documentdb"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+type mongoCollectionInfo = struct {
+	MongoCollection documentdb.MongoDBCollectionGetResults
+	Account         *string
+	Database        *string
+	Name            *string
+	ResourceGroup   *string
+	Location        *string
+}
+
+//// TABLE DEFINITION
+
+func tableAzureCosmosDBMongoCollection(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_cosmosdb_mongo_collection",
+		Description: "Azure Cosmos DB Mongo Collection",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"account_name", "name", "resource_group", "database_name"}),
+			Hydrate:    getCosmosDBMongoCollection,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isNotFoundError([]string{"ResourceNotFound", "NotFound"}),
+			},
+		},
+		List: &plugin.ListConfig{
+			KeyColumns: plugin.KeyColumnSlice{
+				{
+					Name: "database_name", Require: plugin.Required,
+				},
+				{
+					Name: "account_name", Require: plugin.Optional,
+				},
+			},
+			ParentHydrate: listCosmosDBAccounts,
+			Hydrate:       listCosmosDBMongoCollections,
+		},
+		Columns: azureColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Type:        proto.ColumnType_STRING,
+				Description: "The friendly name that identifies the Mongo DB collection.",
+			},
+			{
+				Name:        "account_name",
+				Type:        proto.ColumnType_STRING,
+				Description: "The friendly name that identifies the cosmosdb account in which the collection is created.",
+				Transform:   transform.FromField("Account"),
+			},
```

Comment:
Could you please update the column definition ordering(Name, Description, Type then Transform) to kame all the column definition uniform?